### PR TITLE
Add a new API for GetPlaceStatsVar

### PR DESCRIPTION
**Pull request by shifucun**
This API  fetches all the statistical variable for given places. It helps client to know what stats are available in DataCommons KG for a given place. For example, a web page can use this to determine the available statistical variable and use GetStats API to fetch the actual data.

### CODE REVIEW

File: proto/mixer.proto [hunk 1]
```diff
@@ -408,6 +408,20 @@ message SearchEntityResult {
...
+  repeated string values = 1;
+}
+
+message GetPlaceStatsVarRequest {
+  // DCIDs of the places.
+  repeated string dcids = 1;
+}
+
+message GetPlaceStatsVarResponse {
+  map<string, RepeatedString> places = 8;
```
Comment (Spaceenter) [inline proto/mixer.proto:421]: Why staring from 8?

Comment (shifucun) [reply proto/mixer.proto]: good catch. Updated.

File: e2etest/get_place_statsvar_test.go [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+	}
+
+	dcids := []string{"geoId/05", "geoId/06885", "invalid"}
+	want := map[string]struct {
+		minCount  int
+		statsVars []string
+	}{
+		"geoId/05":    {1000, []string{"TotalPopulation"}},
+		"geoId/06858": {1000, []string{"Person_Count"}},
+		"invalid":     {0, []string{}},
```
Comment (beets) [inline e2etest/get_place_statsvar_test.go:39]: Add a test for an empty/unspecified dcid?

File: e2etest/get_place_statsvar_test.go [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+	}
+	req := &pb.GetPlaceStatsVarRequest{
+		Dcids: dcids,
+	}
+	resp, err := client.GetPlaceStatsVar(ctx, req)
+	if err != nil {
+		t.Errorf("could not GetPlaceStatsVar: %s", err)
```
Comment (beets) [inline e2etest/get_place_statsvar_test.go:46]: Nit: could -> Could

File: e2etest/get_place_statsvar_test.go [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+func TestGetPlaceStatsVar(t *testing.T) {
+	ctx := context.Background()
+	client, err := setup(server.NewMemcache(map[string][]byte{}))
+	if err != nil {
+		t.Fatalf("Failed to set up mixer and client")
+	}
+
+	dcids := []string{"geoId/05", "geoId/06885", "invalid"}
+	want := map[string]struct {
+		minCount  int
```
Comment (beets) [inline e2etest/get_place_statsvar_test.go:34]: Why are we requiring minCount (as opposed to max like in our other API calls)

File: proto/mixer.proto [hunk 5]
```diff
@@ -408,6 +408,20 @@ message SearchEntityResult {
...
+message RepeatedString {
+  repeated string values = 1;
+}
+
+message GetPlaceStatsVarRequest {
+  // DCIDs of the places.
+  repeated string dcids = 1;
+}
+
+message GetPlaceStatsVarResponse {
```
[the remark this answers is not in the document]
Comment (shifucun) [reply proto/mixer.proto:423]: Done

File: e2etest/get_place_statsvar_test.go (hunk 2 above)
Comment (shifucun) [reply e2etest/get_place_statsvar_test.go]: Done

File: e2etest/get_place_statsvar_test.go (hunk 3 above)
Comment (shifucun) [reply e2etest/get_place_statsvar_test.go]: Done

File: e2etest/get_place_statsvar_test.go (hunk 4 above)
Comment (shifucun) [reply e2etest/get_place_statsvar_test.go]: This is to check the returned result has a minimal amount stats vars, mostly for testing the api is working.

Comment (beets) [reply e2etest/get_place_statsvar_test.go]: Ah, ok. It's just for the test..

**Comment by shifucun** [comment]
Added PR description.
